resources/ru/cemu.po

```diff
@@ -1,146 +1,1140 @@
+msgstr "Введите новую инструкцию."
+
+#: DisasmCtrl.cpp DumpCtrl.cpp
+msgid "Enter a target address."
+msgstr "Введите адрес цели."
+
+#: DisasmCtrl.cpp DumpCtrl.cpp
+msgid "GoTo address"
+msgstr "Перейти к адресу"
+
+#: DownloadGraphicPacksWindow.cpp GraphicPacksWindow2.cpp
+msgid "Graphic packs"
+msgstr "Графические пакеты"
+
+#: DownloadGraphicPacksWindow.cpp
+msgid "Graphic packs cannot be updated while a game is running."
+msgstr "Графические пакеты не могут быть обновлены во время работы программы."
+
+#: DownloadGraphicPacksWindow.cpp
+msgid "Failed to connect to server"
+msgstr "Не удалось подключиться к серверу"
+
+#: DownloadGraphicPacksWindow.cpp
+msgid "No updates available."
+msgstr "Нет доступных обновлений."
+
+#: DownloadGraphicPacksWindow.cpp
+msgid ""
+"Updated graphic packs are available. Do you want to download and install "
+"them?"
+msgstr ""
+"Обновлённые графические пакеты доступны. Желаетел ли вы скачать и установить "
+"их?"
+
+#: DownloadGraphicPacksWindow.cpp
+msgid "Checking version..."
+msgstr "Проверка версии..."
+
+#: DownloadGraphicPacksWindow.cpp
+msgid "Downloading graphic packs..."
+msgstr "Загрузка графических пакетов..."
+
+#: DownloadGraphicPacksWindow.cpp
+msgid "Extracting..."
+msgstr "Извлечение..."
+
+#: DumpCtrl.cpp RegisterCtrl.cpp RegisterWindow.cpp
+msgid "Enter a new value."
+msgstr "Введите повое значение."
+
+#: GameProfileWindow.cpp
+msgid "Edit game profile"
+msgstr "Изменить игровой профиль"
+
+#: GameProfileWindow.cpp GeneralSettings2.cpp
+msgid "General"
+msgstr "Основные"
+
+#: GameProfileWindow.cpp
+msgid "Load shared libraries"
+msgstr "Загрузить общие библиотеки"
+
+#: GameProfileWindow.cpp
+msgid ""
+"EXPERT OPTION\n"
+"This option will load libraries from the cafeLibs directory"
+msgstr ""
+"EXPERT OPTION\n"
+"Эта опция загрузит библиотеки из каталога cafeLibs"
+
+# Не совсем корректный перевод, однако передаёт суть происходящего
+#: GameProfileWindow.cpp
+msgid "Launch with gamepad view"
+msgstr "Запуск с окном геймпада"
+
+# Аналогично, не совсем корректно.
+#: GameProfileWindow.cpp
+msgid ""
+"Games will be launched with gamepad view toggled as default. The view can be "
+"toggled with CTRL + TAB"
+msgstr ""
+"Игры будут запущены с включенным по умолчанию окном геймпада. Вид можно "
+"переключить с помощью CTRL + TAB"
+
+#: GameProfileWindow.cpp
+msgid "CPU"
+msgstr "CPU"
+
+#: GameProfileWindow.cpp
+msgid "Mode"
+msgstr "Режим"
+
+#: GameProfileWindow.cpp
+msgid "Singlecore-Interpreter"
+msgstr "Одноядерный интепретатор"
+
+#: GameProfileWindow.cpp
+msgid "Singlecore-Recompiler"
+msgstr "Одноядерный рекомпилятор"
+
+#: GameProfileWindow.cpp
+msgid "Dualcore-Recompiler"
+msgstr "Двухъядерный рекомпилятор"
+
+#: GameProfileWindow.cpp
+msgid "Triplecore-Recompiler"
+msgstr "Трёхъядерный рекомпилятор"
+
+#: GameProfileWindow.cpp
+msgid "Set the CPU emulation mode"
+msgstr "Выберите режим эмуляции CPU"
+
+#: GameProfileWindow.cpp
+msgid "Thread quantum"
+msgstr "Thread quantum"
```

Comment:
This is likely to have a translation in your language. If you're unsure or don't know the exact translation in this context, feel free to leave it as it is